### PR TITLE
Create create_new_project.md

### PR DESCRIPTION
**Pull request by jonathanglassman**
### Context

You need to create new projects.

### Changes proposed in this pull request

New section.

### CODE REVIEW

File: source/manage_project/create_new_project.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    bundle install
+    ```
+
+## View your new project
+
+1. Go to Finder (Mac) or Explorer (Windows) or equivalent.
+
+1. Check for the folder with your project name.
+
+The folder will have multiple pre-populated files. You will only need to amend the `source/documentation` folder and the `config.yml` file.
```
Comment (bahmady) [inline source/manage_project/create_new_project.md]: There's technically no requirement to use a subdirectory `documentation` within `source`. You can just put everything inside other subdirectories with names of your choosing, or even `index.html.md.erb` files as I tend to.

File: source/manage_project/create_new_project.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Create a new project
+
+1. Open the command line interface.
+
+1. Navigate to your documents folder.
+
+1. Run the following in the command line:
+
+    ```
+    Middleman init PROJECT_NAME -T alphagov/tech-docs-template
```
Comment (bahmady) [inline source/manage_project/create_new_project.md]: Not a problem with this PR but as I mentioned elsewhere, we should find a standard for placeholders.

File: source/manage_project/create_new_project.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+1. Run the following in the command line to update all the gems inside the repo:
+
+    ```
+    bundle install
+    ```
+
+## View your new project
+
+1. Go to Finder (Mac) or Explorer (Windows) or equivalent.
```
Comment (bahmady) [inline source/manage_project/create_new_project.md]: We probably shouldn't assume the method of navigation; we could combine these two steps into one about navigating to the directory in question.

File: source/manage_project/create_new_project.md (hunk 1 above)
Comment (jonathanglassman) [reply source/manage_project/create_new_project.md]: agree, have amended

**Pull request merged**